src/plugins/gauge/components/Gauge.vue

```diff
@@ -21,168 +21,259 @@
 *****************************************************************************/
 <template>
 <div
-    class="c-gauge"
+    class="c-gauge__wrapper js-gauge-wrapper"
     :class="`c-gauge--${gaugeType}`"
 >
-    <div class="c-gauge__wrapper">
-        <template v-if="typeDial">
-            <svg
-                class="c-gauge__range"
-                viewBox="0 0 512 512"
-            >
-                <text
-                    v-if="displayMinMax"
-                    font-size="35"
-                    transform="translate(105 455) rotate(-45)"
-                >{{ rangeLow }}</text>
-                <text
-                    v-if="displayMinMax"
-                    font-size="35"
-                    transform="translate(407 455) rotate(45)"
-                    text-anchor="end"
-                >{{ rangeHigh }}</text>
-            </svg>
-
+    <template v-if="typeDial">
```

Comment:
Each gauge type should be a separate component. This issue was [also raised in the original PR but not addressed at the time](https://github.com/nasa/openmct/pull/4919#discussion_r838106544). I'll file a followup.

Reply:
YES. I have unmerged changes in a separate PR (#5083) that also split out the "type" (radial dial vs. horizontal/vertical meter) from the value display (filled area vs. needle). This allows needle-style meters, which we don't have now, and for which VERVE drivers think there is a use case. We should integrate those changes when we address this comment.